src/test/java/de/tum/in/www1/artemis/QuizExerciseIntegrationTest.java

```diff
@@ -31,14 +32,13 @@
 import de.tum.in.www1.artemis.domain.quiz.ShortAnswerQuestion;
 import de.tum.in.www1.artemis.domain.quiz.ShortAnswerSolution;
 import de.tum.in.www1.artemis.domain.quiz.ShortAnswerSpot;
-import de.tum.in.www1.artemis.repository.QuizExerciseRepository;
-import de.tum.in.www1.artemis.repository.ResultRepository;
-import de.tum.in.www1.artemis.repository.StudentParticipationRepository;
+import de.tum.in.www1.artemis.repository.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'de.tum.in.www1.artemis.repository'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968855)

Reply:
See f9e39e2dbe10ffa13483affe2de6359ac2244213